tests/unit/models/test_base_model.py

```diff
@@ -82,15 +83,44 @@ def test_is_dict_of_model_when_not_true(self):
             ),
         )
 
+    def test_bad_type(self):
+        transaction_dict = {
+            "account": 1,
+            "amount": 10,
+            "destination": 1,
+        }
+        with self.assertRaises(XRPLModelException):
+            Payment(**transaction_dict)
+
+    def test_bad_type_flags(self):
+        transaction_dict = {
+            "account": 1,
+            "amount": 10,
+            "destination": 1,
+            "flags": "1234",
+        }
+        with self.assertRaises(XRPLModelException):
+            Payment(**transaction_dict)
+
+    def test_bad_type_enum(self):
+        path_find_dict = {
+            "subcommand": "blah",
```

Comment:
I don't understand this test case. It does not throw an exception if `"subcommand": "create",` --  Hence, the validity rests more on the content of the string, rather than it's data type.


Am I understanding this correctly? Where is this bad enum?

Reply:
The `subcommand` param in the `PathFindResponse` is an enum: https://github.com/XRPLF/xrpl-py/blob/main/xrpl/models/requests/path_find.py#L83

Reply:
ah okay